modules/solid_mechanics/src/materials/ConstitutiveModel.C

```diff
@@ -31,13 +31,13 @@ InputParameters validParams<ConstitutiveModel>()
 }
 
 
-ConstitutiveModel::ConstitutiveModel( const InputParameters & parameters)
+ConstitutiveModel::ConstitutiveModel(const InputParameters & parameters)
   :Material(parameters),
    _has_temp(isCoupled("temp")),
    _temperature(_has_temp ? coupledValue("temp") : _zero),
    _temperature_old(_has_temp ? coupledValueOld("temp") : _zero),
    _alpha(parameters.isParamValid("thermal_expansion") ? getParam<Real>("thermal_expansion") : 0.),
-   _alpha_function( parameters.isParamValid("thermal_expansion_function") ? &getFunction("thermal_expansion_function") : NULL),
+   _alpha_function(parameters.isParamValid("thermal_expansion_function") ? &getFunction("thermal_expansion_function") : NULL),
    _has_stress_free_temp(isParamValid("stress_free_temperature")),
    _stress_free_temp(_has_stress_free_temp ? getParam<Real>("stress_free_temperature") : 0.0),
```

Comment:
I guess I don't understand the logic here. Independent of `_has_stress_free_temp` the first timestep does its thing. Then at the second timestep, where we need the temperature from the first timestep, we now do something different if `_has_stress_free_temp` is set (i.e. **not** take the previous step's temperature). Why?